client/tests/e2e/specs/test_user_details_modal.ts

```diff
@@ -0,0 +1,31 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+describe('Check user details modal', () => {
+  beforeEach(() => {
+    cy.visitApp();
+    cy.login('Boby', 'P@ssw0rd.');
+    cy.get('.organization-card__manageBtn').click();
+    cy.get('.users-container').find('.user-list-item').eq(2).find('.options-button').invoke('show').click();
+    cy.get('.user-context-menu').find('.menu-list').find('ion-item').as('menuItems');
+    cy.get('@menuItems').eq(3).contains('View details').click();
+  });
+
+  afterEach(() => {
+    cy.dropTestbed();
+  });
+
+  it('Tests user details modal', () => {
+    cy.get('.ion-page').find('.ms-modal').as('modal').find('ion-header').contains('User details');
+    cy.get('@modal').find('.ms-modal-content').as('modal-content').should('contain', 'Name').and('contain', 'Joined');
+    // cspell:disable-next-line
+    cy.get('@modal-content').should('contain', 'Jaheira').and('contain', 'one second ago').and('contain', 'Shared workspaces');
```

Comment:
Same here, avoid chaining and be careful with contains. It's fine to use contain on a very specific element, but it's so broad that is you use it on the whole modal, you're not checking much. I wouldn't be surprised if this would match

```
<a href="https://www.lipsum.com/">
Lorem ipsum dolor one second ago sit amet, consectetur adipiscing elit, 
sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi 
ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit 
in voluptate Jaheira velit esse cillum dolore eu fugiat nullaShared 
workspacespariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa 
qui officia deserunt mollit anim id est laborum."</a>
```

when it clearly shouldn't.